role/token_test.go

```diff
@@ -27,9 +27,12 @@ import (
 
 func TestToken_SetParams(t *testing.T) {
 	type fields struct {
+		Principal     string
 		Domain        string
 		Roles         []string
+		IntTimeStamp  int64
 		ExpiryTime    time.Time
+		IntExpiryTime int64
 		KeyID         string
 		Signature     string
 		UnsignedToken string
```

Comment:
```suggestion
		Principal     string
		Domain        string
		Roles         []string
		TimeStamp  int64
		ExpiryTime    time.Time
		KeyID         string
		Signature     string
		UnsignedToken string
```
Please change this test to match the new fields

Reply:
Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/71/commits/95fccb9f7d6f4206dd1617486ac2abdbf7d36edc